articles/monitoring/guides/monitor-applications.md

```diff
@@ -13,20 +13,48 @@ useCase:
   - synthetic-transactions
 ---
 # Monitor Applications
+Auth0 is a critical dependency of your application and/or service. Monitoring Auth0's health can be important as it allows you to report specific errors to your customers and/or take mitigating actions in case of an issue with Auth0.
```

Comment:
in what cases isn't it? We don't want to claim things that aren't true but I can't think of a customer that wouldn't think of auth0 as a critical dependency. Can we acknowledge that?

Reply:
I agree, we are always critical :)

Reply:
Maybe something like:

Your applications and services depend on Auth0. Monitoring Auth0's health lets you report specific errors to customers or reduce user impact in case there's an issue with Auth0.